pl_bolts/datamodules/binary_mnist_datamodule.py

```diff
@@ -136,36 +131,32 @@ def val_dataloader(self, batch_size=32, transforms=None):
         )
         loader = DataLoader(
             dataset_val,
-            batch_size=batch_size,
+            batch_size=self.batch_size,
             shuffle=False,
             num_workers=self.num_workers,
             drop_last=True,
             pin_memory=True
         )
         return loader
 
-    def test_dataloader(self, batch_size=32, transforms=None):
+    def test_dataloader(self):
         """
         MNIST test set uses the test split
-
-        Args:
-            batch_size: size of batch
-            transforms: custom transforms
         """
-        transforms = transforms or self.val_transforms or self._default_transforms()
+        transforms = self.default_transforms() if self.test_transforms is None else self.test_transforms
 
         dataset = BinaryMNIST(self.data_dir, train=False, download=False, transform=transforms)
         loader = DataLoader(
             dataset,
-            batch_size=batch_size,
+            batch_size=self.batch_size,
             shuffle=False,
             num_workers=self.num_workers,
             drop_last=True,
             pin_memory=True
         )
         return loader
 
-    def _default_transforms(self):
+    def default_transforms(self):
```

Comment:
Not sure if we should make this method public...

Reply:
I updated the method to match with the rest of the data modules... Maybe we could get some feedback about the scope from somebody else. `LightningDataModule` abstract class has no actual definition for it so :shrug: 

cc @Borda

Reply:
I think this should just be internal

Reply:
gotcha